planetiler-examples/src/test/java/com/onthegomap/planetiler/examples/BikeRouteOverlayTest.java

```diff
@@ -110,6 +110,8 @@ void integrationTest(@TempDir Path tmpDir) throws Exception {
           "name", "EuroVelo 8 - Mediterranean Route - part Monaco",
           "ref", "EV8"
         ), GeoUtils.WORLD_LAT_LON_BOUNDS, 25, LineString.class);
+
+      TestUtils.assertTileDuplicates(mbtiles, 0);
```

Comment:
Is there somewhere in the core module we can put this? I just want to make sure core functionality is verified there in case we remove/change examples in the future.

Reply:
So far no other UT is triggering the root-cause.

Hence I've tried to make a new UT in `planetiler-core` but then realized I would need to replicate quite a lot of code from `planetiler-examples`.

E.g. both options are "bad".

Conclusion so far was to go with "smaller bad", e.g. the current option, since it adds much smaller amount of code.